### ISSUE TITLE
[BUG] Go install fails with `SECURITY ERROR`

### ISSUE THREAD

**Issue by d-mankowski-synerise** [esnet/gdg#128]
**Describe the bug**

When trying to install gdg via `go install` on Macbook Pro M1, it ends up with `SECURITY ERROR`.

```
❯ go install github.com/esnet/gdg@v0.3.1
go: downloading github.com/esnet/gdg v0.3.1
go: github.com/esnet/gdg@v0.3.1: github.com/esnet/gdg@v0.3.1: verifying module: checksum mismatch
	downloaded: h1:p7FZHrbQS4fSGUnokh/vqGTQxWaNqNrK4B2vJDsKjBk=
	sum.golang.org: h1:bYB31CKs0/FIISMR+yICDOZxjgZdIxSXdYXNEhbf/j4=

SECURITY ERROR
This download does NOT match the one reported by the checksum server.
The bits may have been replaced on the origin server, or an attacker may
have intercepted the download attempt.

For more information, see 'go help module-auth'.

~ took 8s
❯ go version
go version go1.20.2 darwin/arm64
```

**Comment by safaci2000** [comment]
This is probably some issues with release, I'm not sure what caused this.  My guess is that there was some CICD issue and I had to delete the tag and created it.  

Not sure there's anything I can do to fix this, except cut a new release.  v0.3.0 will work if you want to pull that, otherwise, there's a change I'm waiting to be complete and I'll release a 0.3.2 soon if you don't mind bearing with me for another week or two.

**Comment by safaci2000** [comment]
@d-mankowski-synerise Try again, released 0.4.0 So this should no longer be an issue.  

I can confirm on my system at least:

```sh
> go install github.com/esnet/gdg@v0.4.0
go: downloading github.com/esnet/gdg v0.4.0
> gdg version
Build Date:
Git Commit:
Version: 0.4.0
Go Version: go1.20
OS / Arch: darwin amd64
```

**Comment by safaci2000** [comment]
Closing issue, feel free to re-open if it's still a problem.
